### DISCUSSION TITLE
Two Separate Meetings, With Same Name, Seem linked

### DISCUSSION THREAD

**Discussion by Area48Webmaster** [code4recovery/12-step-meeting-list#739] [Q&A]
Hi everyone, we have two meetings with the same name but in different districts and they seem linked by showing on the meeting finder in the same district. When I manually try to correct this through the plugin both meetings change to whatever district I adjusted to. We use airtable to import our meeting information and they are listed as separate districts in that database. Any help would be appreciated!

**Comment by tech2serve** [comment]
Are you saying that in Meeting Guide they are showing as from the same district even though in the plugin you have them assigned to two different districts and two different groups? If so, and we can confirm in your feed they are different/have different slugs, you may need to contact GSO.

A link to your website and the specific meetings in question would me look at this.

Thanks.

**Comment by Area48Webmaster** [comment]
Thanks for getting back to me.

Not exactly, Meeting Guide shows the info correctly and I don't see that they delineate what District meetings are in. In our database, airtable, where we import meeting info, these two meetings are listed in separate (and correct) districts. After refreshing the feed to the plugin these two meetings get listed in the same district. When I adjust the incorrect meeting to the correct district the other meeting also adjusts and now becomes incorrect.

Our website with TSML is: https://aahmbny.org/meetings/?tsml-day=1&tsml-distance=25&tsml-mode=location

The meetings in question are found if you search "Another Chance." Currently they are both listed in District 1, but the meeting in Washington County should be listed as District 13.

Thank you!

**Reply by tech2serve** [reply]
I suspect you've found a bug if you're using our importer, but I'd like to confirm it before I create an Issue for development.

I don't personally use AirTable so I need to hear a little more about your work flow. In particular, tell me your process to import meetings into the plugin. Do you export from AirTable to a csv first, them import into TSML? Or, are you using something else to generate a JSON feed that you then pull into TSML?

And if you able to send me a copy of the table (to devs@code4recovery.org) that might help me duplicate the issue.

Please reply to this thread to make things easier to read and follow along. 

Thanks.

**Reply by 97eighty** [reply]
@Area48Webmaster Or a copy of the URL you're using for the TSML import. That may be helpful. Send to devs@code4recovery.org and reference Discussion #739. Thx

**Comment by Area48Webmaster** [comment]
Thank you both. I'm working with the guy who set up the import process from
AirTable to TSML to better understand our process. It is a JSON, I believe
through Winhost, that pulls from Airtable into TSML. Let me get a little
more info on my end I'll get back to you soon. Thanks for your help and
your patience.

On Wed, Apr 6, 2022 at 5:37 PM 97eighty ***@***.***> wrote:

> @Area48Webmaster <https://github.com/Area48Webmaster> Or a copy of the
> URL you're using for the TSML import. That may be helpful. Send to
> ***@***.*** and reference Discussion #739
> <https://github.com/code4recovery/12-step-meeting-list/discussions/739>.
> Thx
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/739#discussioncomment-2519356>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASKFRH5ZU4DFSFSLZRLGES3VDX723ANCNFSM5RALE46Q>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/739/comments/2519356@
> github.com>
>


-- 
*Matt E.*
Area 48 Technology Chair

**Comment by joshreisner** [comment]
@Area48Webmaster group names (as opposed to meeting names) need to be unique in TSML. try adding a city name or some other difference and they should appear in separate districts
